codecov.yml

```diff
@@ -1 +1,6 @@
-comment: false
\ No newline at end of file
+comment: false
+ignore:
+  - "lib/**/*engine.rb"
```

Comment:
Shouldn't these path include something like `decidim-*`? I'm not sure how this works though!